docs/concepts/testset_generation.md

```diff
@@ -66,8 +66,8 @@ from langchain.chat_models import ChatOpenAI
 # documents = load your documents
 
 # Add custom llms and embeddings
-generator_llm = ChatOpenAI(model="gpt-3.5-turbo")
-critic_llm = ChatOpenAI(model="gpt-4")
+generator_llm = LangchainLLM(llm=ChatOpenAI(model="gpt-3.5-turbo"))
```

Comment:
thank you so much for adding this but could you also import `LangchainLLM` from in line 61 with `from ragas.llms import LangchainLLM`

Reply:
Good catch, just added import statement